docs/cloud-provider-specifics.md

```diff
@@ -155,6 +155,8 @@ To integrate Heptio Ark with GCP, you should follow the instructions below to cr
 
 In the Ark root directory, run the following to first set up namespaces, RBAC, and other scaffolding:
 
+On GKE, be sure to make your current IAM user a cluster-admin. See [their docs][22] for more information.
```

Comment:
s/GKE/GCP yes? (it's what we refer to ^^, and what the link points to)

Reply:
This particular sentence is referring to their Kubernetes Engine product, which is part of GCP's offering. I added this sentence to clarify for GKE users because Ark's prereq file actually errors when applied to GKE clusters.

I was considering making an entirely separate section, but this appears to be the only difference between running one's own cluster on top of GCP virtual machines versus using one generated by their hosted offering so far. It could probably use a little more clarification, though.

Reply:
got it. Maybe leave as is for now and review for clarification later? Up to you ...